docs/en/03_troubleshooting.md

```diff
@@ -1,29 +1,34 @@
+---
+title: Troubleshooting
+---
+
 # Troubleshooting
 
 Check the below if you have any issues during installation or use
 
-## Installation Issues
+## Installation issues
 
 After installation make sure you have done a `dev/build` you may also need to flush the admin view by appending
 `?flush=1` to the URL, e.g. `http://yoursite.com/admin?flush=1`
```

Comment:
```suggestion
`?flush=1` to the URL, e.g. `http://example.com/admin?flush=1`
```
Sorry, I should have caught this last time.

Reply:
Updated